src/lib/controller/incoming_message.ts

```diff
@@ -96,12 +96,20 @@ export interface IncomingCommandControllerRemoveAssociations
   associations: Association[];
 }
 
+// Schema version < 3
 export interface IncomingCommandControllerRemoveNodeFromAllAssocations
   extends IncomingCommandControllerBase {
   command: ControllerCommand.removeNodeFromAllAssocations;
   nodeId: number;
 }
 
+// Schema version > 2
+export interface IncomingCommandControllerRemoveNodeFromAllAssociations
+  extends IncomingCommandControllerBase {
+  command: ControllerCommand.removeNodeFromAllAssociations;
+  nodeId: number;
+}
```

Comment:
Do we want to specify 2 types or do we want to specify 1 interface that represents both commands ?